helpers/type.helpers.ts

```diff
@@ -0,0 +1,78 @@
+import { ETHEREUM_TYPES } from '@/enums'
+import { AddressLike, BytesLike } from '@/types'
+import { isAddress, isBytesLike } from 'ethers'
+import { BigNumber } from 'bignumber.js'
+
+export function checkIsAddress(value: unknown): value is AddressLike {
```

Comment:
checkIsAddress(value: string) and return boolean(you don't need to write a return type)

Reply:
done